Jenkinsfile.revapi

```diff
@@ -0,0 +1,263 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.kie.jenkins.MavenCommand
+
+repos = []
+masterRepos = []
+branched7xRepos = []
+revapiProperties = [:]
+
+pipeline {
+    agent {
+        label 'kie-rhel7'
+    }
+
+    tools {
+        maven 'kie-maven-3.6.3'
+        jdk 'kie-jdk1.8'
+    }
+
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
+        timeout(time: 360, unit: 'MINUTES')
+    }
+
+    parameters {
+        string(name: 'DISPLAY_NAME', defaultValue: '', description: 'Setup a specific build display name')
+
+        // Git information
+        string(name: 'BUILD_BRANCH_NAME', defaultValue: 'master', description: 'Which branch to build? Set if you are not on a multibranch pipeline.')
+        string(name: 'GIT_AUTHOR', defaultValue: 'kiegroup', description: 'Which Git author repository ?')
+        // Revapi branch name
+        string(name: 'REVAPI_RELEASE_VERSION', defaultValue: '', description: 'Release version to apply revapi clean to. E.g. 7.49.Final')
```

Comment:
Here I would change the description so it says it is actually a version revapi should check against. We are applying cleaning always to master.
Also versions should be with the last 0, e.g. `7.49.0.Final`.

Reply:
done